### PR TITLE
[DO NOT MERGE] Refactor accordion component print styles

### PR DESCRIPTION
**Pull request by MartinJJones**
## What / why
- we're removing print stylesheets in favour of using print mixins in the screen stylesheet
- reduces requests, simplifies component management, may reduce asset size in some components
- currently only moving styles from print to screen, not deleting print stylesheet as this would be a breaking change

## Visual Changes
After updating the CSS class names, the accordion toggle buttons no longer show when printing as intended.

| Before | After |
| --- | --- |
| accordion-print-before | accordion-print-after |

Trello card: https://trello.com/c/EHMAHSjh/83-improve-component-print-stylesheets-strategy

### CODE REVIEW

File: app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss [hunk 1]
```diff
@@ -1,7 +1,36 @@
 @import "mixins/prefixed-transform";
 @import "govuk/components/accordion/accordion";
+@import "govuk_publishing_components/component_support";
```
Comment (andysellick) [inline app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss:3]: This is interesting - any idea why `component_support` was previously needed by the print stylesheet? Is it still needed here?

**Review by andysellick — COMMENTED** [review]
Accordion styles look good, just one hopefully small question.

File: app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss (hunk 1 above)
Comment (MartinJJones) [reply app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss:3]: Good catch, it was originally added as part of this commit - https://github.com/alphagov/govuk_publishing_components/commit/1ec93f29deab43bcbdb761670a79e8964483f9af to:

> "Fix all_components_print import, `components/print/accordion` requires `govuk-frontend` mixins as it imports the main `components/accordion`"

None of the other screen stylesheets import `component_support`, as it is expected to be imported already by the application, for example in application.scss - https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/_all_components.scss#L8.

I will remove the import for `component_support`, thanks

Comment (andysellick) [reply app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss:3]: Great, thanks for investigating. Happy to approve now.

**Comment by MartinJJones** [comment]
Closing for https://github.com/alphagov/govuk_publishing_components/pull/3110

**Pull request closed without merging**